app/controllers/v1/fake_providers_controller.rb

```diff
@@ -8,24 +8,48 @@ def show
 
     def provider_details
       {
-        providerOffices: Array.new(rand(1..3)).map { provider_office },
-        contactId: rand(1..100_000),
-        contactName: params[:username]
+        providerOffices: Array.new(number_of_offices) do |office_index|
+          provider_office(office_index)
+        end,
+        contactId: username_number * 3,
+        contactName: contact_name
       }
     end
 
-    def provider_office
+    def provider_office(office_index)
+      # unique office_id for each office
+      office_id = username_number * 2 + office_index
+
+      # unique office_number for each office
+      office_number = "0A#{office_id}"
```

Comment:
I paired with Will on this PR.
This controller is mimicking a CCMS API that returns the list of offices a provider belongs to.
We don't have much information about the format of this office number. Just that some of them are:
```
0R060V
0R442K
0U565T
0U613V
0X512W
```

So we chose to have it start with `0A`. In retrospect, because we don't any real information about the format of this office_number, we can only assume that it's a string.
Maybe this would make more sense and would be less obscure:
```ruby
office_number = "office#{office_id}"
```